client/src/main/kotlin/io/spine/examples/pingh/client/DesktopClient.kt

```diff
@@ -88,28 +88,22 @@ internal class DesktopClient(
             .getOrNull(0)
 
     /**
-     * Observes both events until one is emitted.
+     * Observes events until one is emitted.
```

Comment:
What you have written here is read like the following: by calling this method, we observe all events, until any is emitted. This is simply not what we do.

Please rephrase.